yaml/connector-dep.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: vcenter
+    component: vcenter-connector
+  name: vcenter-connector
+  namespace: openfaas
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: vcenter
+        component: vcenter-connector
+    spec:
+      containers:
+      - name: vcenter
+        image: functions/vcenter-connector:latest
+        env:
+          - name: vcenter-url
```

Comment:
This needs to be an "arg" - see the openfaas-operator or the of-builder / of-buildkit YAML files as an example of how to pass args.